### PR TITLE
added tests to verify /variant and /sitemap  API behind akamai

### PR DESCRIPTION
**Pull request by pbadak**

### CODE REVIEW

File: suites/test_module_content.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
 import suites.test_module_edit_publish as test_module_edit_publish
 import os
 
+PROXIES = {'http': 'http://squid.corp.redhat.com:3128', 'https': 'http://squid.corp.redhat.com:3128'}
```
Comment (noopurmath3) [inline suites/test_module_content.py:15]: This can be read from the config.ini file
It is an existing value

**Review by noopurmath3 — COMMENTED** [review]
Added 2 comments
Other than that LGTM!

File: suites/test_module_content.py (hunk 1 above)
Comment (anarang) [inline suites/test_module_content.py:15]: @pbadak please move the proxy config to config.ini, actual values don't go here!

**Review by anarang — CHANGES_REQUESTED** [review]

File: suites/test_module_content.py [hunk 2]
```diff
@@ -21,15 +23,14 @@
...
-
 @lcc.suite(description="Suite: Verify contents of published module", rank=2)
 class test_module_content:
   api_auth = lcc.inject_fixture("api_auth")
 
   path_for_module = ""
   request_url = ""
 
-  @lcc.test("Verify response of module variant api")
+  @lcc.test("Verify response of module variant api for pantheon API")
```
[the remark this answers is not in the document]
Comment (pbadak) [reply suites/test_module_content.py]: done

**Review by noopurmath3 — APPROVED** [review]
LGTM :+1:

**Pull request merged**